Content.Server/SS220/Storage/SpawnOnOpen/Components/SpawnItemsOnOpenComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Robust.Shared.Prototypes;
+using Content.Shared.EntityTable.EntitySelectors;
+
+namespace Content.Server.SS220.Storage.SpawnOnStorageOpen.Components;
+
+[RegisterComponent]
+public sealed partial class SpawnOnStorageOpenComponent : Component
+{
+    /// <summary>
+    ///     Basically any class inheriting from <see cref="EntityTableSelector"/> will fit
+    /// </summary>
+    [DataField("selector")]
+    public EntityTableSelector Selector;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("uses")]
```

Comment:
`DataField` по умолчанию даёт права `ReadWrite` для VV
```suggestion
    [DataField]
```